CITATION.cff

```diff
@@ -0,0 +1,51 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: TopoStats
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Alice
+    email: a.l.pyne@sheffield.ac.uk
+    affiliation: The University of Sheffield
+    orcid: 'https://orcid.org/0000-0002-2658-8987'
+    family-names: ' Pyne'
```

Comment:
Family-name seems out of order and also contains quotes.

Reply:
This file was generated as per the comments at the top, copy and pasted it direct. No doubt reflects the order in which the fields were completed. From memory I first typed `Alice Pyne` in the first field then copy and pasted the surname to the correct field and will have inadvertently included the space. The quotes will have been added because of the presence of a space.